src/r2mm/manager/GameRunner.ts

```diff
@@ -10,17 +9,37 @@ import Profile from '../../model/Profile';
 
 export default class GameRunner {
 
+    public static chooseExecutable(): string {
+        switch (process.platform) {
+            case 'win32':
+                return 'Steam.exe';
+            case 'linux':
+                return 'steam.sh';
+            default:
+                return 'steam';
+        }
+    }
+
     public static async playModded(ror2Directory: string, onComplete: (err: R2Error | null) => void) {
         LoggerProvider.instance.Log(LogSeverity.INFO, 'Launching modded');
         const settings = await ManagerSettings.getSingleton();
         const steamDir: string | R2Error = await GameDirectoryResolver.getSteamDirectory();
+        const preloaderDir: string =
```

Comment:
Close, but not quite.
The `BepInEx.Preloader.dll` is stored in the `<profile>/BepInEx/core/` directory. This will always be resolvable unless BepInEx has not been installed, in which case the command will be ignored.

This has nothing to do with Wine/Proton.

Reply:
Trust me it needs to be there to work properly.

You're used to this kind of scenario where you pass the path of BepInEx as an argument and call it a day; and it would work fine if we're strictly talking Windows.

On Linux, every Windows game is ran in a sandbox and you cannot just pass Linux paths to it. The `path` module alone won't ever return a sandbox-compatible path, and that is why I needed to add this constant and prefix the path with Z: if we're not on Windows, again, because
- The sandbox behaves like Windows would, if we're seeing it from the game's perspective
- The *nix root file system is mounted at Z: in the sandbox
- The *nix file system doesn't have letters as mountpoints, but it has the root (/) and subsequent directories

Remember a key thing: r2modman will run natively on Linux so it has to account for Wine/Proton. This has nothing to do with the DLL itself existing or not.